webroot/js/view/list.js

```diff
@@ -386,6 +394,28 @@ export class ListView extends View {
 		if (this.editing && this.tbody) {
 			return;
 		}
+
+		// render at most once every 1000 msec
```

Comment:
really nessasary? - the gui should `render()` on new messages by websocket 
https://github.com/FreifunkBremen/freifunkmanager/blob/485ecc13c686a1819df8b5e6a013dc9189a904a2/webroot/js/socket.js#L65-L102

Reply:
If many updates are received in a short time, the page is re-rendered very often. With this commit the page is only re-rendered at most once per second, but not more. This reduce CPU load for the browser.

Reply:
A got Point,
I would re implement render BOUNCING again:
https://github.com/FreifunkBremen/freifunkmanager/blob/37cf186d320ee852de827c3978dc8315f4f0368f/webroot/js/gui.js#L87-L100

Reply:
Ah, du hattest sowas schonmal eingebaut :-)
Bei der Bouncing-Variante wird aber renderView() nicht nochmal aufgerufen, wenn der Timeout abgelaufen ist, oder? Dadurch bleiben evtl. alte Werte stehen.